src/coord/src/catalog.rs

```diff
@@ -1426,10 +1437,10 @@ pub enum Op {
 #[derive(Debug, Clone)]
 pub enum OpStatus {
     CreatedDatabase(String, i64),
-    CreatedSchema,
+    CreatedSchema(String, String, i64),
     CreatedItem(GlobalId),
     DroppedDatabase(String, Option<i64>),
-    DroppedSchema,
+    DroppedSchema(String, String, Option<i64>),
```

Comment:
This is weird—it's weird for `DroppedDatabase` too, now that I look at it. We should not include the `OpStatus` entry at all if the database or schema doesn't exist, I think, rather than including the name but not the ID.

Reply:
Sounds good! I will create a clean up PR to fix this for `DroppedDatabases` next.